content/docs/dvclive/ml-frameworks/pytorch-lightning.md

```diff
@@ -61,6 +61,23 @@ checkpointing at all as described in the
 
 - `prefix` - (`None` by default) - string that adds to each metric name.
 
+- `log_model` - (`False` by default) - use
+  [`live.log_artifact()`](/doc/dvclive/live/log_artifact) to log checkpoints
```

Comment:
log checkpoints - does it means that we might have multiple records? per checkpoint?